### ISSUE TITLE
[EN] [NPC] [Grizzly Hills] Ruuna the Blind

### ISSUE THREAD

**Issue by lourrific** [Atlantiss/BugTracker#3634]
http://www.wowhead.com/npc=27581/ruuna-the-blind

This NPC does not offer me the quest Out of Body Experience http://www.wowhead.com/quest=12327/out-of-body-experience

I have done all the prerequisites that should lead me up to this quest.
They are:

1. Descent into Darkness
2. Report to Gryan Stoutmantle... Again
3. Hollowstone Mine
4. Souls at Unrest
5. A Name from the Past
6. Ruuna the Blind 
7. Ruuna's Request

As a result of not being able to do the quest Out of Body Experience http://www.wowhead.com/quest=12327/out-of-body-experience I can not aquire the following quests:

1. Fate and Coincidence
2. Anatoly Will Talk
3. Sasha's Hunt (optional sidequest) 
4. A Sister's Pledge
5. Hour of the Worg 

So basically 5 quests that I can not do ruining the Grizzly Hills progression Achievement. http://www.wowhead.com/achievement=37/fo-grizzle-my-shizzle

**Comment by Avalain** [comment]
Last time this bug was reported it was fixed: #2526

"krecik88 commented on 9 Mar 2015
fixed in 4354"

After some time quest stopped appearing for both Horde and Alliance.

**Comment by podatekvat** [comment]
5418 hash 
It's still bugged. She doesn't offer Out of Body Experience http://www.wowhead.com/quest=12327/out-of-body-experience

**Comment by Samineh** [comment]
Bump :+1:
